templates/standards/nist-csf/v1.1.json

```diff
@@ -0,0 +1,577 @@
+{
+  "standard": "NIST CSF 1.1",
```

Comment:
@henrymgarrett pls use this file path in CV2 so that we can deprecate/remove the one in the parent directory